### ISSUE TITLE
Error using Logtail config in django

### ISSUE THREAD

**Issue by akshay-codemonk** [logtail/logtail-python#8]
Logging configuration in settings.py

```
LOGGING = {
    "version": 1,
    "disable_existing_loggers": False,  # },
    'handlers': {
        'logtail': {
            'class': 'logtail.LogtailHandler',
            'source_token': LOGTAIL_SOURCE_TOKEN,
        },
    },
    "loggers": {
        "django": {
            "handlers": [
                "logtail",
            ],
            "level": "INFO",
            "propagate": False,
        },
        "apps": {
            "handlers": [
                "logtail",
            ],
            "level": "INFO",
            "propagate": False,
        },
    },
}

```

Error on console, during logging

```
Traceback (most recent call last):
server_1         |   File "/usr/local/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
server_1         |     self.run()
server_1         |   File "/usr/local/lib/python3.10/site-packages/logtail/flusher.py", line 23, in run
server_1         |     self.step()
server_1         |   File "/usr/local/lib/python3.10/site-packages/logtail/flusher.py", line 60, in step
server_1         |     response = self.upload(frame)
server_1         |   File "/usr/local/lib/python3.10/site-packages/logtail/uploader.py", line 18, in __call__
server_1         |     data = msgpack.packb(frame, use_bin_type=True)
server_1         |   File "/usr/local/lib/python3.10/site-packages/msgpack/__init__.py", line 38, in packb
server_1         |     return Packer(**kwargs).pack(o)
server_1         |   File "msgpack/_packer.pyx", line 294, in msgpack._cmsgpack.Packer.pack
server_1         |   File "msgpack/_packer.pyx", line 300, in msgpack._cmsgpack.Packer.pack
server_1         |   File "msgpack/_packer.pyx", line 297, in msgpack._cmsgpack.Packer.pack
server_1         |   File "msgpack/_packer.pyx", line 264, in msgpack._cmsgpack.Packer._pack
server_1         |   File "msgpack/_packer.pyx", line 231, in msgpack._cmsgpack.Packer._pack
server_1         |   File "msgpack/_packer.pyx", line 291, in msgpack._cmsgpack.Packer._pack
server_1         | TypeError: can not serialize 'socket' object

```

Versions used: 
python-3.10
Django==3.2.15
logtail-python==0.1.4

**Comment by mirzadelic** [comment]
I am having the same issue, is there a solution for this?

**Comment by curusarn** [comment]
Hi @mirzadelic and @akshay-codemonk,

Thanks again for bringing this up!

Django passes a request object to the log handler as part of the logging context.
I have released a new version [v0.2.2](https://github.com/logtail/logtail-python/releases/tag/v0.2.2) with a fix.
It filters out the request if it's not serializable.

Could you update to v0.2.2 and let me know if it solves your issue? 🙏

**Comment by curusarn** [comment]
I'm closing this since the issue was fixed and should no longer happen.

Feel free to reopen if you are still encountering the issue.
